cookbooks/fb_systemd/recipes/default_packages.rb

```diff
@@ -59,6 +59,9 @@
        !['trusty', 'jessie'].include?(node['lsb']['codename'])
       systemd_packages << 'systemd-journal-remote'
     end
+    if node['fb_systemd']['timesyncd']['enable'] && node.debian?
```

Comment:
this is not runtime safe, you cannot read the node at compile time, no one will have had a chance to set it yet.

We should just always install it.

We COULD make a separate package resource and use an `only_if` here, if there's a good reason, but our default is to install all the relevant software unless `manage_packages` is `false`.

Reply:
I updated this to always install timesyncd if it is enabled on all platforms:
```
if node['fb_systemd']['timesyncd']['enable']
```

I believe it is safe to read this node attribute since there is a `lazy` block wrapping everything: https://github.com/facebook/chef-cookbooks/blob/main/cookbooks/fb_systemd/recipes/default_packages.rb#L56

but I'm happy to move this package elsewhere in this file! it felt funny always installing it since someone could not choose to disable `timesyncd`.

Reply:
thanks for the speedy feedback on this PR btw!